tests/libraries/token/test_standard_token.py

```diff
@@ -36,4 +36,86 @@ def test_eternal_approval_magic(contractsFixture):
     assert standardToken.allowance(tester.a0, tester.a1) == 2 ** 256 - 1
     assert standardToken.balanceOf(tester.a0) == 4
     assert standardToken.balanceOf(tester.a1) == 96
-    
\ No newline at end of file
+    
+def test_create_negative_balance(contractsFixture):
+    standardToken = contractsFixture.uploadAndAddToController("solidity_test_helpers/StandardTokenHelper.sol")
+    assert standardToken.totalSupply() == 0
+    # We get some tokens for tester 0
+    assert standardToken.faucet(100)
+    assert standardToken.balanceOf(tester.a0) == 100
+    # Tester 1 tries to send tokens from tester 0 to himself and fails
+    with raises(TransactionFailed):
+        standardToken.transferFrom(tester.a0, tester.a1, 101)
```

Comment:
`transferFrom` in general, is only used if you are trying to send someone else's tokens.  If you want to send your own tokens you would use `transfer`.  If you want to specify who the sender is you can do `standardToken.transferFrom(tester.a1, 101, sender=tester.k0)`.  In this particular case, `tester.k0` is the default so you could leave it out.

Reply:
ok, removed the unnecessary tester.k0 and changed the transferFrom()  to transfer()